conf/.htaccess.assets.expiry

```diff
@@ -0,0 +1,9 @@
+# BEGIN core-standards:assetsExpiry
+<IfModule mod_expires.c>
+  # Expires + Cache-Control header with a TTL of 1 year for all assets.
+  <FilesMatch "\.(?:css|cur|eot|gif|gz|htc|ico|jpe?g|js|mp4|ogg|ogv|otf|png|svgz?|ttf|webm|woff2?)\?ver">
```

Comment:
1. "ver" is not guaranteed to be the only query string parameter, so this regex needs some adjustment. You should also add the terminating = to prevent false-positives on other parameter names.

2. The exclusion for local hostnames is missing? (see issue summary)